opensrp-chw/src/kk/resources/anc_hv_danger_signs.properties

```diff
@@ -20,7 +20,7 @@ anc_hv_danger_signs.step1.danger_signs_present.options.chk_signs_of_malaria.text
 anc_hv_danger_signs.step1.danger_signs_present.options.chk_difficulty_breathing.text = Fast/difficult breathing
 anc_hv_danger_signs.step1.danger_signs_present.options.chk_convulsions.text = Convulsions/Fits
 anc_hv_danger_signs.step1.danger_signs_present.options.chk_none.text = None
-anc_hv_danger_signs.step1.danger_signs_present_toaster.text = Danger sign! \nYou need to go to the closest health facility immediately. It is important that you complete the referral. I will follow-up with you within 24 hours to check if you have completed your referral.
+anc_hv_danger_signs.step1.danger_signs_present_toaster.text = I have identified an urgent Danger sign! \nYou need to go to the closest health facility immediately. It is important that you complete the referral. I will follow-up with you within 24 hours to check if you have completed your referral.
```

Comment:
Can you also change this in the swahili translation?

Reply:
Done